packages/amplify-graphql-docs-generator/__tests__/generator/utils/loading.test.ts

```diff
@@ -0,0 +1,26 @@
+import { loadSchema } from '../../../src/generator/utils/loading';
+
+describe('loading', () => {
+  it('JSON schema', () => {
+    const graphqlSchema = loadSchema(__dirname + '/../../../fixtures/schema.json');
+    expect(graphqlSchema.constructor.name).toBe('GraphQLSchema');
+  });
+
+  it('JSON schema without data property', () => {
+    const graphqlSchema = loadSchema(__dirname + '/../../../fixtures/schemaWithoutDataProperty.json');
+    expect(graphqlSchema.constructor.name).toBe('GraphQLSchema');
+  });
+  it('JSON schema file is not exits', () => {
```

Comment:
Please add a newline between the tests. Same thing between lines 21 and 22.

Reply:
Thanks for your review.
I fixed it.